heroic-dist/src/test/java/com/spotify/heroic/AbstractClusterQueryIT.java

```diff
@@ -599,7 +652,7 @@ private void testAggregationLimit(final MetricType metricType, final String quer
 
 
     private void validateStatAccuracy(final Map<Long,Double> resMap){
-        // validate that error rate is less than 2/100 for each datapoint.
```

Comment:
did this functionality change?

Reply:
No, I made it better 1/100 instead of 2/100 because we are using uniform distributed data.  https://github.com/spotify/heroic/pull/744/files/628c4c5166b62540f3f7dfe2f83011cbf24606d5#diff-e16773fcf30192fba5132f1732fd45848a8638a392947e222f3a1979fa9a1fde

Reply:
ah i see. could have left the comment with an updated value but not a big deal